plugins/right-sizer/src/controller/controller.go

```diff
@@ -162,6 +171,10 @@ func NewController(stop chan struct{}, kubeClientResources util.KubeClientResour
 func (c *Controller) Run() error {
 	glog.Infof("Controller configuration is: %+v", c.config)
 	glog.Infof("Report configuration is: %s", c.reportBuilder.GetConfigAsString())
+	dissimilarNamespaces := funk.LeftJoinString(c.config.allowedUpdateNamespaces, c.config.allowedNamespaces)
+	if c.config.updateMemoryLimits && len(dissimilarNamespaces) > 0 {
+		glog.Errorf("NOTE: memory limits will not be updated in these Kubernetes namespaces: %v Although these namespaces are in the allowedUpdateNamespaces list, they are missing from the global allowedNamespaces one.", dissimilarNamespaces)
+	}
```

Comment:
I'm not sure I agree with this check - why not just do everything in `allowedUpdateNamespaces`?

Also - don't we have to worry about the case where `allowedNamespaces` is empty (meaning all namespaces)?

Reply:
> I'm not sure I agree with this check - why not just do everything in `allowedUpdateNamespaces`?
Do you not agree with the option in its entirety (a separate namespace filter only applying to updating limits), or where the check is in the code?

> Also - don't we have to worry about the case where `allowedNamespaces` is empty (meaning all namespaces)?
Yes, I also realized this earlier today - fixed. Thanks for catching this!